js/freewheel.js

```diff
@@ -315,7 +315,7 @@ OO.Ads.manager(() => {
      * Freewheel assumes that the ad video element is an html5 video tag.  To force use of this element,
      * always list the stream type as mp4.
      * @method Freewheel#_prepareTimeline
-     * @returns {null|array} Returns [] or null
+     * @returns {null|array} [] or null
```

Comment:
it seems that we don't use the result of this function, so there's no need to return anything, just `void`